components/forecast/AvalancheTab.tsx

```diff
@@ -53,88 +52,67 @@ export const AvalancheTab: React.FunctionComponent<AvalancheTabProps> = React.me
 
   const elevationBandNames: ElevationBandNames = zone.config.elevation_band_names;
 
+  const [textColor] = useToken('colors', ['darkText']);
```

Comment:
What's this dark/light switch for? Is it using the built-in dark-mode support in Native Base?

Reply:
this is for grabbing the default text color from the NativeBase theme